### ISSUE TITLE
[Help] Start server error

### ISSUE THREAD

**Issue by bigdogs** [schemathesis/schemathesis#1049]
Hello, 

When I exec `bash test_server.sh`， it report error "cannot import name 'FrozenList' from 'graphql.pyutils'",
![image](https://user-images.githubusercontent.com/20614926/111281094-87a68700-8677-11eb-8fea-e46ea5b9b6c1.png)

After some investigation, I found the dependency conflict:
`graphene 2.1.8` requires `graphql-core` version < 3
`graphql-server` requires `graphal-core` version > 3

I have no idea how to fix this issue

**Comment by Stranger6667** [comment]
Hi @bigdogs 

What steps did you take? It seems like your local dependencies do not match the project's requirements.

Schemathesis test server requires `graphene = "^3.0b3"` as per [dev dependencies list](https://github.com/schemathesis/schemathesis/blob/master/pyproject.toml#L55).

Here is the output of executing the steps listed in [the local test server guide] on my local machine (https://github.com/schemathesis/schemathesis/blob/master/CONTRIBUTING.rst#using-a-local-test-server) after a fresh git clone:

```
example-schemathesis on  master via 🐍 v3.9.2 (example-schemathesis)
➜ poetry install       
Installing dependencies from lock file

Package operations: 64 installs, 0 updates, 0 removals

  • Installing pyparsing (2.4.7)
  • Installing attrs (20.3.0)
  • Installing iniconfig (1.1.1)
  • Installing packaging (20.9)
  • Installing pluggy (0.13.1)
  • Installing py (1.10.0)
  • Installing toml (0.10.2)
  • Installing apipkg (1.5): Installing...
  • Installing certifi (2020.12.5): Installing...
  • Installing chardet (4.0.0): Installing...
  • Installing apipkg (1.5)
  • Installing certifi (2020.12.5)
  • Installing chardet (4.0.0)
  • Installing graphql-core (3.1.3)
  • Installing idna (2.10)
  • Installing markupsafe (1.1.1)
  • Installing multidict (5.1.0)
  • Installing pyrsistent (0.17.3)
  • Installing pytest (6.2.2)
  • Installing pytz (2021.1)
  • Installing six (1.15.0)
  • Installing sortedcontainers (2.3.0)
  • Installing urllib3 (1.26.3)
  • Installing alabaster (0.7.12): Installing...
  • Installing aniso8601 (8.1.1): Installing...
  • Installing async-timeout (3.0.1): Installing...
  • Installing babel (2.9.0): Installing...
  • Installing click (7.1.2): Installing...
  • Installing docutils (0.16): Installing...
  • Installing alabaster (0.7.12)
  • Installing aniso8601 (8.1.1)
  • Installing async-timeout (3.0.1)
  • Installing babel (2.9.0): Installing...
  • Installing click (7.1.2)
  • Installing docutils (0.16): Installing...
  • Installing execnet (1.8.0)
  • Installing graphql-relay (3.1.0)
  • Installing click (7.1.2)
  • Installing docutils (0.16): Installing...
  • Installing execnet (1.8.0)
  • Installing graphql-relay (3.1.0)
  • Installing babel (2.9.0)
  • Installing click (7.1.2)
  • Installing docutils (0.16): Installing...
  • Installing execnet (1.8.0)
  • Installing graphql-relay (3.1.0)
  • Installing hypothesis (6.7.0): Installing...
  • Installing execnet (1.8.0)
  • Installing graphql-relay (3.1.0)
  • Installing hypothesis (6.7.0): Installing...
  • Installing docutils (0.16)
  • Installing execnet (1.8.0)
  • Installing graphql-relay (3.1.0)
  • Installing hypothesis (6.7.0): Installing...
  • Installing hypothesis (6.7.0)
  • Installing imagesize (1.2.0)
  • Installing itsdangerous (1.1.0)
  • Installing jinja2 (2.11.3)
  • Installing jsonschema (3.2.0)
  • Installing pydantic (1.7.3)
  • Installing pygments (2.8.1)
  • Installing pytest-forked (1.3.0)
  • Installing requests (2.25.1)
  • Installing snowballstemmer (2.1.0)
  • Installing sphinxcontrib-applehelp (1.0.2)
  • Installing sphinxcontrib-devhelp (1.0.2)
  • Installing sphinxcontrib-htmlhelp (1.0.3)
  • Installing sphinxcontrib-jsmath (1.0.1)
  • Installing sphinxcontrib-qthelp (1.0.3)
  • Installing sphinxcontrib-serializinghtml (1.1.4)
  • Installing starlette (0.13.6)
  • Installing typing-extensions (3.7.4.3)
  • Installing werkzeug (1.0.1)
  • Installing yarl (1.6.3)
  • Installing aiohttp (3.7.4.post0): Installing...
  • Installing coverage (5.5): Installing...
  • Installing curlify (2.2.1): Installing...
  • Installing aiohttp (3.7.4.post0)
  • Installing coverage (5.5)
  • Installing curlify (2.2.1)
  • Installing fastapi (0.62.0)
  • Installing flask (1.1.2)
  • Installing graphene (3.0b7)
  • Installing graphql-server-core (2.0.0 e8f3a89)
  • Installing hypothesis-jsonschema (0.19.0)
  • Installing junit-xml (1.9)
  • Installing pytest-asyncio (0.11.0)
  • Installing pytest-mock (1.13.0)
  • Installing hypothesis-graphql (0.3.3)
  • Installing pytest-subtests (0.4.0)
  • Installing pytest-xdist (1.34.0)
  • Installing pyyaml (5.4.1)
  • Installing sphinx (3.5.2)

Installing the current project: schemathesis (3.2.2)
example-schemathesis on  master via 🐍 v3.9.2 (example-schemathesis)took 10s 
➜ ./test_server.sh 8081
Schemathesis test server is running!

API Schema is available at: http://0.0.0.0:8081/schema.yaml

DEBUG:asyncio:Using selector: EpollSelector
======== Running on http://0.0.0.0:8081 ========
(Press CTRL+C to quit)
```

Which works as expected.

**Comment by bigdogs** [comment]
hello, 

Thanks, `poetry` works :)
